admin_panel/src/commonMain/kotlin/net/thechance/mena/admin_panel/presentation/screen/mainContainer/component/AdminSideBarTabs.kt

```diff
@@ -0,0 +1,110 @@
+
+@Composable
+fun AdminSideBarTabs(
+
+    selectedTab: MainContainerScreenState.CurrentTab,
+    onTabSelected: (tab: MainContainerScreenState.CurrentTab) -> Unit,
+    modifier: Modifier = Modifier
+) {
+    Column(
+        verticalArrangement = Arrangement.spacedBy(24.dp),
+        horizontalAlignment = Alignment.CenterHorizontally,
+        modifier = modifier
+    ) {
+        AdminSidebarItem(
+            title = stringResource(MainContainerScreenState.CurrentTab.DUKAN_MANAGEMENT.title),
+            selectedIcon = painterResource(MainContainerScreenState.CurrentTab.DUKAN_MANAGEMENT.selectedIconRes),
+            notSelectedIcon = painterResource(MainContainerScreenState.CurrentTab.DUKAN_MANAGEMENT.unSelectedIconRes),
+            isSelected = selectedTab == MainContainerScreenState.CurrentTab.DUKAN_MANAGEMENT,
+            onClick = { onTabSelected(MainContainerScreenState.CurrentTab.DUKAN_MANAGEMENT) }
+        )
+
+        AdminSidebarItem(
+            title = stringResource(MainContainerScreenState.CurrentTab.DUKAN_REQUEST.title),
+            selectedIcon = painterResource(MainContainerScreenState.CurrentTab.DUKAN_REQUEST.selectedIconRes),
+            notSelectedIcon = painterResource(MainContainerScreenState.CurrentTab.DUKAN_REQUEST.unSelectedIconRes),
+            isSelected = selectedTab == MainContainerScreenState.CurrentTab.DUKAN_REQUEST,
+            onClick = { onTabSelected(MainContainerScreenState.CurrentTab.DUKAN_REQUEST) }
+        )
+
+        AdminSidebarItem(
+            title = stringResource(MainContainerScreenState.CurrentTab.DEPOSIT.title),
+            selectedIcon = painterResource(MainContainerScreenState.CurrentTab.DEPOSIT.selectedIconRes),
+            notSelectedIcon = painterResource(MainContainerScreenState.CurrentTab.DEPOSIT.unSelectedIconRes),
+            isSelected = selectedTab == MainContainerScreenState.CurrentTab.DEPOSIT,
+            onClick = { onTabSelected(MainContainerScreenState.CurrentTab.DEPOSIT) }
+        )
+
+        AdminSidebarItem(
+            title = stringResource(MainContainerScreenState.CurrentTab.USERS_MANAGEMENT.title),
+            selectedIcon = painterResource(MainContainerScreenState.CurrentTab.USERS_MANAGEMENT.selectedIconRes),
+            notSelectedIcon = painterResource(MainContainerScreenState.CurrentTab.USERS_MANAGEMENT.unSelectedIconRes),
+            isSelected = selectedTab == MainContainerScreenState.CurrentTab.USERS_MANAGEMENT,
+            onClick = { onTabSelected(MainContainerScreenState.CurrentTab.USERS_MANAGEMENT) }
+        )
+    }
+}
+
+@Composable
+private fun AdminSidebarItem(
+    title: String,
+    selectedIcon: Painter,
+    notSelectedIcon: Painter,
+    isSelected: Boolean,
+    onClick: () -> Unit
+) {
+    val animatedTitleColor by animateColorAsState(
+        targetValue =
+            if (isSelected)
+                Theme.colorScheme.shadeSecondary
+            else
+                Theme.colorScheme.brand.brand
+    )
```

Comment:
We can remove the if condition outside animateColorAsState for easier readability
```suggestion
    val titleColor = if (isSelected) Theme.colorScheme.shadeSecondary else Theme.colorScheme.brand.brand
    
    val animatedTitleColor by animateColorAsState(
        targetValue = titleColor
    )
```